### ISSUE TITLE
[femdomcult] offline

### ISSUE THREAD

**Issue by ilike2burnthing** [Jackett/Jackett#13365]
### Description

https://femdomcult.org/
- ERR_CONNECTION_REFUSED
- Fatal error: Uncaught PDOException
- HTTP ERROR 500
- default UNIT3D login page

http://femdomcult.org/
- blank page
- 502 Bad Gateway
- loads Empornium home page

**Comment by garfield69** [comment]
@ilike2burnthing  The login page is back. Working?

**Comment by ilike2burnthing** [comment]
No. Tried once (with definitely correct credentials) and got:

> 403 Your IP is banned for another 6 hours
You have insufficient privileges to access the requested page or resource.

Both the recover and register pages say they send emails, but nothing is ever received.

**Comment by ilike2burnthing** [comment]
http://torrentinvites.org/f31/femdomcult-news-606640/
https://www.reddit.com/r/trackers/comments/vvgumb/best_femdomcult_alternative/

TL;DR - server issues, repeated attempts to fix it failed, changed engine several times, one user has said that the torrent tracker is back online as of a few days ago.

I'd say give it to the end of the month.

**Comment by ilike2burnthing** [comment]
Slow as hell, but it's back up.

**Comment by ilike2burnthing** [comment]
back up

**Comment by ilike2burnthing** [comment]
https://femdomcult.org/ - ERR_CONNECTION_TIMED_OUT

**Comment by garfield69** [comment]
online

**Comment by ilike2burnthing** [comment]
Are you able to sign in? When I try I just get the `Maintenance Mode` notice again.

**Comment by garfield69** [comment]
ah. i don't have an active account, but when i do used my disabled creds i too get the maint message
